dts/bindings/dali/nxp,dali.yaml

```
@@ -0,0 +1,41 @@
+---
+description: DALI interface for nxp lpc11u6x
+
+compatible: "nxp,dali"
```

This `compatible` feels too generic? Is it a HW peripheral of a single SoC?

```suggestion
compatible: "nxp,dali-lpc11u6x"
```

---

Absolutely, everything else is very specific for the chip and so should the binding. I changed the binding and all references to it.